src/Sets/HalfSpace.jl

```diff
@@ -529,15 +529,15 @@ return quote
 # returns `(true, sexpr)` if expr represents a half-space,
 # where sexpr is the simplified expression sexpr := LHS - RHS <= 0
 # otherwise, returns `(false, expr)`
-function _is_halfspace(expr::Operation)
+function _is_halfspace(expr::Term)
     got_halfspace = true
 
     # find sense and normalize
-    if expr.op == <
+    if (expr.op == <=) || (expr.op == <)
```

Comment:
```suggestion
    if expr.op in (<=, <)
```
This is just an aesthetic suggestion. Feel free to ignore it.